src/elements/content-sidebar/AddTaskMenu.js

```diff
@@ -28,7 +28,11 @@ const AddTaskMenu = (props: Props) => (
             </MenuToggle>
         </Button>
         <Menu className="bcs-AddTaskMenu">
-            <MenuItem className="bcs-AddTaskMenu-menuItem" onClick={() => props.onMenuItemClick(TASK_TYPE_GENERAL)}>
+            <MenuItem
+                className="bcs-AddTaskMenu-menuItem"
+                data-target-id="MenuItem-generaltask"
```

Comment:
I noticed that `camelCase` notation is mostly used for the second part of `data-target-id `across the repository. Is there any specific reason why this notation isn't being followed here?

Reply:
Yes, we use this structure for `data-target-id`. I contacted Kamil and he shared the examples.

Reply:
While working with Babel plugin I noticed that it uses camel case ids like this: `data-target-id="Button-deleteAssetText"`.
So I'm wondering why the structure Kamil shared doesn't match.
Let me confirm it.

Reply:
I can confirm that with the previous addition of `data-target-id` attr manually, I copied the naming from `data-testid` and it was not camelCase. However I agree that we should use camelCase here and in the future as this is mostly how it's named automatically by Babel plugin.

Reply:
Thanks for confirming, I'll update it to camel case!

Reply:
Thank you